packages/measured-core/lib/metrics/Meter.js

```diff
@@ -22,6 +22,10 @@ class Meter {
   constructor(properties) {
     this._properties = properties || {};
     this._initializeState();
+
+    if (!this._properties.keepAlive) {
```

Comment:
What are the consequences of making this the default behavior rather than inverting this logic?
Is this a breaking change for people?

Reply:
I mean having the default be false.

Reply:
In any application that has an active handle (e.g. listening on a port, etc) it's usually not the measured Timer/Meter interval timer is keeping the event loop alive. I see it as highly unlikely to be breaking.

If you see this as a problem I can certainly unpublish 1.43.0 and release this as a major but to me this very much is a minor.